detox/src/devices/LaunchArgs.js

```diff
@@ -0,0 +1,32 @@
+const _ = require('lodash');
+
+class LaunchArgs {
+  constructor(launchArgs = {}) {
+    this._args = launchArgs;
+  }
+
+  modify(launchArgs) {
+    Object
+      .keys(launchArgs)
+      .forEach((name) =>
+        this._setLaunchArg(name, launchArgs[name]));
+  }
+
+  reset() {
+    this._args = {};
```

Comment:
Wait, wait — are we resetting to an empty object or to the original `launchArgs`? This looks like a discrepancy.